tests/phpunit/elementor/core/files/test-manager.php

```diff
@@ -4,8 +4,17 @@
 use Elementor\Core\Files\Manager;
 use Elementor\Core\Page_Assets\Data_Managers\Base as Page_Assets_Data_Manager;
 use ElementorEditorTesting\Elementor_Test_Base;
+use WP_REST_Request;
```

Comment:
The global classes do not need to be imported here; just use `\WP_REST_Request`.